aioxmpp/muc/service.py

```diff
@@ -39,6 +39,21 @@
 from . import xso as muc_xso
 
 
+def _extract_one_pair(body):
```

Comment:
This is a bit problematic. What if the client on the other side is rude and includes empty messages for some language tags, this would severely degrade the tracking.